README.md

```diff
@@ -432,6 +432,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 * [Doctrine](http://www.doctrine-project.org/) - A comprehensive DBAL and ORM.
 * [Doctrine Extensions](https://github.com/l3pp4rd/DoctrineExtensions) - A collection of Doctrine behavioural extensions.
 * [Propel](http://www.propelorm.org/) - A fast ORM, migration library and query builder.
+* [LazyRecord](http://github.com/c9s/LazyRecord) - A fast ORM that builds your schema into static class files and offers you a smarter automatic migration mechanism.
```

Comment:
Could this be shortened? The official repo's description is and nicer (no marketing buzz please). :wink: